cmd/mothership/mothership/start/http.go

```diff
@@ -238,7 +237,8 @@ func startWebserver(ctx context.Context, o *Options) error {
 func enableReconciliationDebugLogging(o *Options, w http.ResponseWriter, r *http.Request) {
 
 	if !features.Enabled(features.DebugLogForSpecificOperations) {
-		w.WriteHeader(http.StatusNotFound)
+		err := fmt.Errorf("enabling debug logs for a reconciliation is disabled")
```

Comment:
In my opinion , it's better to use `errors.New` instead of` fmt.Errorf `because the error in this line is not a custom error message.

Reply:
Yes. that's definitely better. I have applied it! thanks for the suggestion 👍